lib/dogma/rules/final_newline.ex

```diff
@@ -8,7 +8,9 @@ defmodule Dogma.Rules.FinalNewline do
   alias Dogma.Script
   alias Dogma.Error
 
-  def test(script) do
+  def test(script), do: test(script, [])
+
+  def test(script, _config = []) do
```

Comment:
I considered this but then I didn't want it to match:
```elixir
test(script, banana: "not_valid_config_key")
```
I wonder if it is it possible/sensible to do
```elixir
def test(script, _config = [] \\ [])?
```

Thoughts @lpil ?